app/models/work_activity_notification.rb

```diff
@@ -3,4 +3,10 @@
 class WorkActivityNotification < ApplicationRecord
   belongs_to :work_activity
   belongs_to :user
+
+  after_create do
```

Comment:
This seems untested.  I may have missed it though...

Reply:
I am verifying this right at this moment, however, I believe that https://github.com/pulibrary/pdc_describe/pull/484/files#diff-9860398629ad665126bc447cc0b151e236450de156e8bc64a313c18db62bc7c3R8 should cover this.